recipe/build.sh

```diff
@@ -2,14 +2,5 @@
 
 set -e
 
-mkdir -p $PREFIX/bin
-
-cd prodigal_modified
-make CC=$CC # conda will add $CC to environment
-chmod +x prodigal
-cd ..
-
-cp $SRC_DIR/prodigal_modified/prodigal $PREFIX/bin/prodigal_sm
-
+$PYTHON -m pip install pyrodigal>=0.7.3
```

Comment:
Why is this not installed with `conda`/`mamba`?

Reply:
moved to the install script